FlowCal/excel_ui.py

```diff
@@ -170,13 +172,27 @@ def write_workbook(filename, table_list, column_width=None):
     # Pandas' default header format is bold text with thin borders. Here we
     # use bold text only, without borders.
     # The format module is in pd.core.format in pandas<=0.18.0,
-    # pd.formats.format in pandas>=0.18.1.
+    # pd.formats.format in pandas>=0.18.1, and pd.io.formats.excel in
+    # pandas>=0.20.
```

Comment:
Lol yeah the code works. It has been tested with 0.18.0, 0.18.1,, 0.19.1, 0.20.1, and 0.20.2. I will update the description.